hassio/dns.py

```diff
@@ -215,15 +222,28 @@ def _write_corefile(self) -> None:
             _LOGGER.error("Can't read coredns template file: %s", err)
             raise CoreDNSError() from None
 
-        # Prepare DNS serverlist: Prio 1 Local, Prio 2 Manual, Prio 3 Fallback
+        # Prepare DNS serverlist: Prio 1 Manual, Prio 2 Local, Prio 3 Fallback
+        # If Manual servers are identical to Fallback (was default before 194), then ignore manual
         local_dns: List[str] = self.sys_host.network.dns_servers or ["dns://127.0.0.11"]
+        if DNS_SERVERS == self.servers:
```

Comment:
That now not happens anymore with your change above. So you can fallback to your first solution

Reply:
Got it, thanks - removed condition in 1a8c1d5